tests/test_linalg.py

```diff
@@ -115,3 +115,74 @@ def test_matrix_rank():
             result = inp.linalg.matrix_rank(ia, tol=tol)
             expected = numpy.linalg.matrix_rank(a, tol=tol)
             numpy.testing.assert_array_equal(expected, result)
+
+
+def test_norm1():
+    arrays = [
+        [7],
+        [1, 2],
+        [1, 0]
+    ]
+    ords = [None, -numpy.Inf, -2, -1, 0, 1, 2, 3, numpy.Inf]
+    axis_ = [0, None]
+    for array in arrays:
+        for ord in ords:
+            for axis in axis_:
```

Comment:
Please, redesigned this test suit. Designed it as a parametrize test function, use  see: https://docs.pytest.org/en/latest/example/parametrize.html
See also some examples from our test cases: https://github.com/IntelPython/dpnp/blob/master/tests/test_logic.py#L41:#L48